core/creator/src/main/java/io/quarkus/creator/phase/augment/AugmentTask.java

```diff
@@ -177,14 +180,17 @@ public AugmentOutcome run(CurateOutcome appState, CuratedApplicationCreator ctx)
                 builder.addFinal(BytecodeTransformerBuildItem.class)
                         .addFinal(ApplicationArchivesBuildItem.class)
                         .addFinal(MainClassBuildItem.class)
-                        .addFinal(ArtifactResultBuildItem.class);
+                        .addFinal(ArtifactResultBuildItem.class)
+                        .addFinal(DockerImageResultBuildItem.class)
+                        .addFinal(DeploymentResultBuildItem.class); //This should probably go to a different class. io.quarkus.deployer.DeployerTask, maybe?
```

Comment:
We need to have a bit more of a think about how to handle deployment, we certainly don't want to deploy after every build.

Reply:
Of course, in the current POC deployments are only triggered when using `-Dquarkus.container.deployment=true`.